strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/indexing/local/RepositoryHostedIndexCreator.java

```diff
@@ -165,12 +174,11 @@ private ArtifactEntryArtifactContextHelper createArtifactContextHelper(final Art
             return new ArtifactEntryArtifactContextHelper(pomExists, sourcesExists, javadocExists);
         }
 
-        for (final ArtifactEntry neighbour : group)
+        for (final Artifact neighbour : group)
         {
             final MavenArtifactCoordinates neighbourCoordinates = (MavenArtifactCoordinates) neighbour.getArtifactCoordinates();
-            pomExists |=
-                    ("pom".equals(neighbourCoordinates.getExtension()) &&
-                     neighbourCoordinates.getClassifier() == null);
+            pomExists |= ("pom".equals(neighbourCoordinates.getExtension()) &&
+                    neighbourCoordinates.getClassifier() == null);
```

Comment:
Cosmetic: Could be a one-liner:
```suggestion
            pomExists |= ("pom".equals(neighbourCoordinates.getExtension()) && neighbourCoordinates.getClassifier() == null);
```